eva/udfs/pytorch_abstract_udf.py

```diff
@@ -23,18 +23,9 @@
 from eva.udfs.gpu_compatible import GPUCompatible
 from eva.configuration.configuration_manager import ConfigurationManager
 
-try:
-    import torch
-    from torch import nn, Tensor
-except ImportError as e:
-    raise ImportError(f"Failed to import with error {e}, \
-        please try `pip install torch`")
-
-try:
-    from torchvision.transforms import Compose, transforms
-except ImportError as e:
-    raise ImportError(f"Failed to import with error {e}, \
-        please try `pip install torchvision`")
+import torch
```

Comment:
Rather add test case for this

Reply:
have we decided to make `torch` optional?

Reply:
@xzdandy @pchunduri6 What do you guys think?

Reply:
Right now. `pip` uses fixed version of dependency (https://github.com/georgia-tech-db/eva/blob/master/setup.cfg). Will this become a problem when users install their own torch version?

Reply:
I actually removed the torch installation from setup.cfg and had to install it separately anyway for ensuring compatible cuda version.